sdk/data/azcosmos/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos
 
-go 1.18
+go 1.21
```

Comment:
This will fail CI as we run versions N, N-1 and 1.18 (N-2) as our baseline. We need to get 1.22 into CI which will resolve the first two parts. For N-2, you'll need to update your CI file to exclude it. See example [here](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/tracing/azotel/ci.yml#L28).

Reply:
FYI CI has been updated. If you rebase/merge latest you'll see 1.22 and 1.21 as N and N-1 versions.